Finding.Object.Game.html

```diff
@@ -0,0 +1,261 @@
+#Score {
+  border: 2px solid red;
+  padding: 5px;
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+  font-size: 20px;
+  color: #009900;
+}
+.DistanceToNearest {
+  color: #0033FF;
+}
+.NumberOfObjects {
+ color: #0033FF;
+}
+.navigation {
+  position: absolute;
+  bottom: 10px;
+  right: 10px;
+}
+.arrow1 {
+  position: absolute;
+  font-size: 50px;
+  width: 60px;
+  height: 60px;
+  display: block;
+  bottom: 100px;
+  right: 30px;
+  color: green;
+}
+.arrow2 {
+  position: absolute;
+  font-size: 50px;
+  width: 60px;
+  height: 60px;
+  display: block;
+  bottom: 24px;
+  color: #ff1a40;
+  right: 30px;
+}
+.arrow3 {
+  position: absolute;
+  font-size: 50px;
+  width: 60px;
+  height: 60px;
+  display: block;
+  bottom: 60px;
+  right: 70px;
+  color: green;
+}
+#Time {
+ position: absolute;
+ bottom: 20px;
+ right: 490px;
+ font-size: 20px;
+ color: red;
+}
+.arrow4 {
+  position: absolute;
+  font-size: 50px;
+  width: 60px;
+  height: 60px;
+  display: block;
+  bottom: 60px;
+  right: 7px;
+  color: #ff1a40;
+}
+.NumberOfObjects {
+   position: absolute;
+   top: 10px;
+   right: 10px;
+   font-size: 20px;
+}
+#divX {
+   position: absolute;
+   right: 20px;
+   bottom: 300px;
+   font-size: 20px;
+}
+#divY {
+   position: absolute;
+   right: 20px;
+   bottom: 275px;
+   font-size: 20px;
+}
+.MyGame {
+   position: absolute;
+   right: 540px;
+   font-size: 26px;
+   top: 5px;
+}
+body {
+   background-color: yellow	;
+   font-family: "Bauhaus 93";
+}
+meter {
+  width: 200px;
+  height: 25px;
+  border-radius: 10px;
+}
+.winner { 
+    background-image: url('http://www.pngall.com/wp-content/uploads/2016/04/Winner-PNG-File.png');
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: center; 
+}
+<!---.monster {
+  background-image: url('http://vignette4.wikia.nocookie.net/looneytunes/images/3/34/Sylvester_Cat_Cartoon-psd12937.png');
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  right: 40px;
+  background-position: 200px 100px;
+  top: 100px;
+  background-size: 150px;
+}-->
+</style>
+<script type="text/javascript">
+    var nbObject = 0;
+    var score=0;
+	var won = false;
+	var hidden_objects = [];
+	var start_time;
+
+    function init() {
+      document.onkeydown = onKeyPress;
+      var score_elt = document.getElementById("Score");
+	  score_elt.setAttribute("value","0");
+      nbObjects = document.getElementById("NbObjects").value;
+      createHiddenObjects(nbObjects);
+      score = 0;
+      start_time = (new Date()).getTime() / 1000;
+      score_elt.innerHTML = 'Score  = ' + score;
+      
+    }
+    
+    function NbObjectUpdate() { // When user change the Number of Objects field
+      nbObjects = document.getElementById("NbObjects").value;
+	  createHiddenObjects(nbObjects);
+	  start_time = (new Date()).getTime() / 1000;
+      var y = Number(document.getElementById("y").value);
+      var x = Number(document.getElementById("x").value);
```

Comment:
What are the Divs `x` and `y` for? Do they have any visual element to them?

Reply:
Yes, this was used for the coordinates part of the game to help the player know where they are.